aces/idt/core/common.py

```diff
@@ -756,3 +758,153 @@ def format_exposure_key(key: str) -> str:
         return f"+{key_float:g}"
     else:
         return f"{key_float:g}"
+
+
+def get_clipping_threshold() -> float:
```

Comment:
We should compute this one as a global once, e.g.,

`CLIPPING_THRESHOLD = 2 / (2**10 - 1)`

It is a bit overkill to write a function for this, especially when the value never changes.

Reply:
Yeah fair its easy enough to document this as a constant and not need the variable names etc. I'll make this a constant